src/testUtils/java/uk/gov/hmcts/reform/waworkflowapi/api/CreatorUtils.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.waworkflowapi.api;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class CreatorUtils {
```

Comment:
Maybe call it `CreatorObjectMapper`?
I just have a dislike of things called `Utils` as then end up becoming a bucket of unrelated stuff over time.